openc3-redis/users.acl

```diff
@@ -2,4 +2,4 @@ user healthcheck on nopass -@all +cluster|info +ping
 user openc3 on >openc3password allkeys allchannels -@all +@read +@write +@pubsub +@connection +@transaction +info
 user scriptrunner on >scriptrunnerpassword resetkeys resetchannels ~running-script* ~*script-locks ~*script-breakpoints ~*openc3_log_messages &_action_cable_internal &script-api:* -@all +@read +@write +@pubsub +@hash +@connection
 user admin on >adminpassword +@admin
-user default off
+user traefik on >traefikpassword allchannels ~traefik* -@all +@admin +@read +@pubsub
```

Comment:
Do we want to keep the `user default off`?

Reply:
Yes. default user is all powerful.